db/active_replicator_checkpointer.go

```diff
@@ -58,12 +58,16 @@ type Checkpointer struct {
 type statusFunc func(lastSeq string) *ReplicationStatus
 
 type CheckpointerStats struct {
-	ExpectedSequenceCount     int64
-	ProcessedSequenceCount    int64
-	AlreadyKnownSequenceCount int64
-	SetCheckpointCount        int64
-	GetCheckpointHitCount     int64
-	GetCheckpointMissCount    int64
+	ExpectedSequenceCount           int64
+	ExpectedSequenceLen             int
+	ExpectedSequenceLenPostCleanup  int
```

Comment:
I went back and forth on it, but I'm fine with the extra stats, even just for test usage.  They are inexpensive and add decent value from a testability perspective.